gammapy/utils/random.py

```diff
@@ -30,6 +30,14 @@ def sample_sphere(size, lon_range=None, lat_range=None, unit='radians'):
     lon, lat: arrays
         Longitude and latitude coordinate arrays
     """
+    # Correctly interpret units at an early stage
+    if unit in ['rad', 'radian', 'radians']:
```

Comment:
@mapazarr – Only people with write access to repos can add labels and assign issues on Github. For full-text search, just go to https://github.com/gammapy/gammapy and type in your question in the search field. Here's an example: https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=observation

The Github search is awesome, but there are cases where it doesn't do what you naively expect. In those cases, or when you're unsure how to best perform a search, head over here: https://help.github.com/categories/search/